index.html

```diff
@@ -23,7 +23,7 @@
       }
 
       /* Link inside h1 (ohm logo/title) */
-      h1>a {
+      h1 > a {
```

Comment:
Not a big deal, but these whitespace only changes (because you ran prettier on the code) should be in a separate commit.